chain-signatures/node/src/web/mock.rs

```diff
@@ -36,6 +36,22 @@ impl MockServer {
             .create_async()
             .await;
 
+        server
+            .mock("GET", "/status")
+            .with_status(201)
+            .with_header("content-type", "text/plain")
```

Comment:
wouldn't it be better to use application/json here?

```suggestion
            .with_header("content-type", "application/json")
```

Reply:
I guess mockito doesn't make use of this at all then